src/PictureCard/index.js

```diff
@@ -0,0 +1,230 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import Text from "../Text";
+import Heading from "../Heading";
+import Stack from "../Stack";
+import defaultTheme from "../defaultTheme";
+import { BASE_URL, SMALLEST_HEIGHT } from "./consts";
+import LazyImage from "../LazyImage";
+import { left } from "../utils/rtl";
+import randomID from "../utils/randomID";
+import KEY_CODE_MAP from "../common/keyMaps";
+
+import type { Props } from "./index";
+
+const Shown = styled.div`
+  transition: all ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  opacity: 0;
+  position: relative;
+  top: "-50%";
+  margin-bottom: ${({ theme }) => theme.orbit.spaceXXSmall};
+`;
+
+Shown.defaultProps = {
+  theme: defaultTheme,
+};
+
+const overlayCss = css`
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  transition: opacity ${({ theme }) => theme.orbit.durationNormal} ease-in-out;
+`;
+
+const StyledLabel = styled.div`
+  position: absolute;
+  top: 0;
+  ${left}: 0;
+  z-index: 2;
+  padding: ${({ theme }) => theme.orbit.spaceMedium};
+`;
+
+StyledLabel.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledOverlay = styled.div`
+  ${overlayCss};
+  opacity: 1;
+  background: linear-gradient(
+    to ${left},
+    rgba(0, 0, 0, 0) 0%,
+    rgba(0, 0, 0, 0.1) 16%,
+    rgba(0, 0, 0, 0.4) 44%,
+    rgba(0, 0, 0, 0.8) 100%
+  );
+`;
+
+StyledOverlay.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledOverlayHover = styled.div`
+  ${overlayCss};
+  opacity: 0;
+  background: rgba(0, 0, 0, 0.8);
+`;
+
+StyledOverlayHover.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledPictureCardContent = styled.div`
+  position: relative;
+  z-index: 2;
+  display: flex;
+  flex-direction: column;
+  padding: ${({ theme }) => theme.orbit.spaceMedium};
+  transition: bottom ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  bottom: ${({ contentHeight, theme }) =>
+    contentHeight && `-${contentHeight + parseInt(theme.orbit.spaceMedium, 10)}px`};
+`;
+
+StyledPictureCardContent.defaultProps = {
+  theme: defaultTheme,
+};
+
+const StyledPictureCard = styled(({ height, href, theme, external, ...props }) => {
+  const Component = href ? "a" : "div";
+  return <Component {...props}>{props.children}</Component>;
+})`
+  height: ${({ height }) => (height ? `${height}px` : "100%")};
+  width: ${({ width }) => (width ? `${width}px` : `100%`)};
+  display: flex;
+  align-items: flex-end;
+  position: relative;
+  box-sizing: border-box;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  overflow: hidden;
+  cursor: pointer;
+  box-shadow: ${({ theme }) => theme.orbit.boxShadowAction};
```

Comment:
The default box-shadow should be visible only when `onClick` or `href` specified. There can be use case when we want to have the inner transition (the actions shows up), but user needs to explicitly click on the content inside it.